ra2ce/graph/network_config_wrapper.py

```diff
@@ -75,8 +64,10 @@ def from_data(
         _new_network_config.ini_file = ini_file
         _new_network_config.config_data = config_data
         if config_data.output_graph_dir and config_data.output_graph_dir.is_dir():
-            _new_network_config.files = _new_network_config.get_existent_network_files(
-                config_data.output_graph_dir
+            _new_network_config.graph_files = (
+                _new_network_config.get_existent_network_files(
+                    config_data.output_graph_dir
+                )
```

Comment:
Why doing this instead of directly mapping the values we just set?

Reply:
```python
        if config_data.graph_files.has_graphs():
            _new_network_config.graph_files = config_data.graph_files
        elif config_data.output_graph_dir and config_data.output_graph_dir.is_dir():
            _new_network_config.graph_files = (
                _new_network_config.get_existent_network_files(
                    config_data.output_graph_dir
                )
            )
        else:
            logging.error(
                f"Graph dir not found. Value provided: {config_data.output_graph_dir}"
            )

```

Reply:
Will look into this separately (take your PR)